cmudb/tscout/model.py

```diff
@@ -677,3 +677,13 @@ def __init__(self):
 
         self.operating_units = operating_units
         self.metrics = OU_METRICS
+        self._enums = nodes.enum_map
+
+    def get_enum_value_map(self, enum_name):
+        if enum_name not in self._enums.keys():
+            logger.error('Requested enum \"%s\" not in PostgreSQL code base.' % enum_name)
+            return {}
+
+        return {
+            entry[0]: entry[1] for entry in self._enums.get(enum_name, [])
```

Comment:
Since you already checked up above, you should never reach the default `[]` case.

```suggestion
            entry[0]: entry[1] for entry in self._enums[enum_name]
```